### PR TITLE
Pravega: Renaming builder method

### PR DESCRIPTION
**Pull request by cheleb**
References #xxxx

### CODE REVIEW

File: pravega/src/main/scala/akka/stream/alpakka/pravega/PravegaSettings.scala [hunk 1]
```diff
@@ -479,7 +482,7 @@ class TableWriterSettingsBuilder[K, V](
   /**
     Build the settings.
    */
-  def withSerializers(
+  def withKeyExtractor(
       extractor: K => TableKey
   ): TableWriterSettingsBuilder[K, V] = copy(tableKeyExtractor = Some(extractor))
```
Comment (ennru) [inline pravega/src/main/scala/akka/stream/alpakka/pravega/PravegaSettings.scala:487]: Something is off here. The comment should go on the `build()` method.
`withSerializers` seems to have been the wrong name, right, that's why you just rename it now?

Comment (cheleb) [reply pravega/src/main/scala/akka/stream/alpakka/pravega/PravegaSettings.scala:487]: Or just be removed !

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by ennru** [comment]
Thank you!

**Pull request merged**